faunadb-java/src/test/java/com/faunadb/client/ClientSpec.java

```diff
@@ -1180,18 +1180,19 @@ public void shouldEvalFormatExpression() throws Exception {
     );
   }
 
-  @Test
-  public void shouldEvalContainsExpression() throws Exception {
-    Value contains = query(
-      Contains(
-        Path("favorites", "foods"),
-        Obj("favorites",
-          Obj("foods",
-            Arr(Value("crunchings"), Value("munchings")))))
-    ).get();
-
-    assertThat(contains.to(BOOLEAN).get(), is(true));
-  }
+// NOT AVAILABLE WITH API V4
```

Comment:
@nathaniel-may, we had this ticket for taking care of `Contains` removal: [PROD-745](https://faunadb.atlassian.net/browse/PROD-745). Summer put a comment there, asking when that should happen. I remember from our meetings that we said that it will be deprecated (API v3) and then removed in the following version (API v4), right? That seems to be what has been done in core (because it's not working anymore!), but I wanted to check if that was still the deprecation strategy! :)

If that's case, maybe we should work on and merge [PROD-745](https://faunadb.atlassian.net/browse/PROD-745) first?

Reply:
I have reinstated the `Contains` test as Marrony moved it deprecated https://github.com/fauna/faunadb-jvm/pull/263#discussion_r501725209

Reply:
Perfect! 👍 As discussed with @nathaniel-may, it looks like `Contains` will be effectively removed in a later version.